client/my-sites/plans-comparison/plans-comparison-action.tsx

```diff
@@ -88,7 +92,7 @@ export const PlansComparisonAction: React.FunctionComponent< Props > = ( {
 		manageHref = undefined;
 	}
 
-	if ( ! isInSignup && [ TYPE_FLEXIBLE, TYPE_FREE ].includes( plan.type ) ) {
+	if ( ! isInSignup && [ TYPE_FLEXIBLE, TYPE_FREE, TYPE_STARTER ].includes( plan.type ) ) {
```

Comment:
This will not show the `Choose Starter` button on the Plans page if you're on a Free plan site.
<img width="320" alt="Screenshot on 2022-05-11 at 16-45-30" src="https://user-images.githubusercontent.com/2749938/167864876-3ebfb08e-f11f-45a0-8d10-ba39417f37d9.png">

Is this intentional?

Reply:
but let's cc @southp -> free plans can upgrade to Starter right?